File: src/com/dafttech/terra/game/world/tiles/TileAir.java
package com.dafttech.terra.game.world.tiles;

import com.badlogic.gdx.graphics.g2d.TextureRegion;
import com.dafttech.terra.resources.Resources;

public class TileAir extends Tile {

    @Override
    public TextureRegion getImage() {
        return Resources.TILES.getImage("air");
    }

    @Override
    public boolean isAir() {
        return true;
    }
}

File: terra/src/main/scala/com/dafttech/terra/game/world/tiles/TileAir.scala
package com.dafttech.terra.game.world.tiles

import com.badlogic.gdx.graphics.g2d.TextureRegion
import com.dafttech.terra.resources.Resources
import monix.eval.Task

class TileAir extends Tile {
  override val getImage: Task[TextureRegion] = Resources.TILES.getImageTask("air")

  override def isAir: Boolean = true
}